lib/fe/maybe.ex

```diff
@@ -103,7 +103,7 @@ defmodule FE.Maybe do
 
   @doc """
   Applies first element from the provided list and the value of the provided
-  `FE.Maybe` to the provided function, that should return a `FE.Maybe`.
+  `FE.Maybe` (base case) to the provided function, that should return a `FE.Maybe`.
```

Comment:
Hm, I don't get the "base case" bit here. What do you mean?

Reply:
I guess it would be easier just to reword the docs so that they mention an accumulator. A base case here is just an initial accumulator.

Reply:
Makes sense 👍